### ISSUE TITLE
Edit mode zoom in/out by mouse scroll.

### ISSUE THREAD

**Issue by velmafia** [ribtoks/xpiks#525]
Hi,
in Edit mode we can see two size of editing images, adjust - default, and full size, will be nice to use mouse scroll to zoom in/out images, and mouse pointer to move images left-right/up-down. 

aloha!

**Comment by ribtoks** [comment]
Yes, I agree that would be nice. Looking forward for somebody to volunteer to implement this.
